spikeinterface/core/segmentutils.py

```diff
@@ -264,5 +267,43 @@ def get_unit_spike_train(self, *args, **kwargs):
 def append_sortings(*args, **kwargs):
     return AppendSegmentSorting(*args, **kwargs)
 
-
 append_sortings.__doc__ == AppendSegmentSorting.__doc__
+
+# Q: should this take as input a concatenated recording or just a list of recordings?
```

Comment:
I think we should have 3 options:
- concatenated recording
- recording_list
- None: in this case we can check if the sorting has a registered recording and if that is a concatenated one

Reply:
Done but please see my changes and comment